src/questions/aave-other-features.yaml

```diff
@@ -109,5 +109,169 @@
+  answerKeys:
+    - A
+    - B
+    - D
+  subTopics:
+    - aave-portals
+  difficultyLevel: "Medium"
+  choices:
+    - content: By reducing liquidity limitations of l2-l2 bridges, the portal will allow for a seamless experience for users
+      key: A
+    - content: Portals make alt Evm L1 chains attractive and easier to access
+      key: B
+    - content: There is no way to confirm the trustworthiness of the validators who help transfer assets in portals
+      key: C
+    - content: Portal's feature seamlessly allows assets to flow between Aave markets on different networks
+      key: D
+
+- uuid: 4f30b2d2-a588-4997-b978-69ce6b00e3cb
+  type: SingleChoice
+  content: Aave portals technology makes use of bridges (T/F)
+  hint: noHint
+  explanation: Aave portals are just a feature to transfer assets efficiently, but we need bridging protocols to transfer our assets. There are no other core protocols for directly using Aave portals from the user end.
+  answerKeys:
+    - A
+  subTopics:
+    - aave-portals
+  difficultyLevel: "Medium"
+  choices:
+    - content: |
+       True
+      key: A
+    - content: |
+       False
+      key: B
+
+- uuid: 860f56ac-1c8f-4522-a576-343a93ab9b80
+  type: SingleChoice
+  content: What are aTokens?
+  hint: noHint
+  explanation: Aave’s version of an already existing crypto currency. They are minted and burnt upon supply and withdrawal of the original currency. For example, aDai.
+  answerKeys:
+    - C
+  subTopics:
+    - aave-portals
+  difficultyLevel: "Medium"
+  choices:
+    - content: aTokens are tokens given to user as incentive for being liquidity provider
+      key: A
+    - content: Aave's version of an already existing currency whose value is more than the original currency
+      key: B
+    - content: Aave’s version of an already existing crypto currency, minted and burnt upon supply and withdrawal of the original currency. 
+      key: C
+    - content: It is a new crypto currency that is independant of any other crypto currency
+      key: D
+
+- uuid: e6751153-9dcd-4e45-873e-c47a7da030a7
+  type: MultipleChoice
+  content: How are aave portals beneficial to the DeFi/ Blockchain ecosystem?
+  hint: noHint
+  explanation: Provides a more secure, faster and cheaper alternative to blockchain bridges, making flow of assets across blockchains easier. Helps increase usership of EVM Layer 1 chains. Helps increase the 
+   user experience of the blockchain network by providing fast and scalable solutions to their issues related to transactions. 
+  answerKeys:
+    - B
+    - C
+    - D
+  subTopics:
+    - aave-portals
+  difficultyLevel: "Medium"
+  choices:
+    - content: They provide a completely new protocol for users to be able to interact with different chains
+      key: A
+    - content: Provides a more secure, faster and cheaper alternative to blockchain bridges, making flow of assets across blockchains easier
```

Comment:
Since portals use existing bridges that are not `more secure` as compared to bridges.